doc/source/install.rst

```diff
@@ -16,4 +16,26 @@ To install the latest development version, do
 
 NOTE: This version is considered UNSTABLE. DON'T use it in the production environment!
 
-Stable version will be released soon.
\ No newline at end of file
+Stable version will be released soon.
+
+Since we are using `plpython3` in Greenplum/Postgres to better use GreenplumPython we also need
+to install the package in the server.
```

Comment:
rephrase to:

GreenplumPython requires `plpython3` extension to be installed on Greenplum/Postgres.

Reply:
copy that